howso/utilities/installation_verification.py

```diff
@@ -422,26 +421,24 @@ def generate_dataframe(*, client: AbstractHowsoClient,
         }
     }
     feature_names = list(features.keys())
-    context_features = list(feature_names)[:-1]
-    action_features = [list(feature_names)[-1]]
 
-    trainee_obj = Trainee(
-        f"installation_verification generated dataframe ({get_nonce()})",
+    trainee = dict(
```

Comment:
Lets just pass these as params directly to the method call, in all instances in this file